packages/components/src/components/button/_button.scss

```diff
@@ -213,7 +213,7 @@
     padding: $button-padding-sm;
   }
 
-  &.#{$prefix}--btn--field {
+  .#{$prefix}--btn--field {
```

Comment:
`--btn--field` is used elsewhere without `&` so from what I can see it's unnecessary anyway.

Reply:
@emyarod Would you agree?

Reply:
looks like it was introduced in https://github.com/carbon-design-system/carbon/pull/2719/

I've tested without `&` and it seems unnecessary